Synergy/Frontend/src/app/features/register-team/components/dropdown-components/football-standard-dropdown/football-standard-dropdown.component.html

```diff
@@ -0,0 +1,34 @@
+<hlm-menu-bar class="w-full hover:cursor-pointer">
+  <button id="toggleButton" [brnMenuTriggerFor]="role" class="w-full gap-3 pl-3 hover:cursor-pointer" hlmMenuBarItem>{{ chosenStandard }}
+    <i
+    *ngIf="chosenStandard !== 'Professional'" class="fa-solid fa-futbol text-blue-800"></i>
```

Comment:
The standard and the age thing is something that I think we can improve on and make it more scalable going forward, remind me to go over this with you